assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -113,11 +101,7 @@ const drawContainingRow = function(container, store, siteno, parameterCode) {
         }, getSelectedParameterCode))
         .on('click', function(event) {
             // Don't let clicks on the sampling method selections trigger a parameter reload.
-            if (isMethodSelectionClick(event.target)) {
-                return;
-            }
-
-            select('#primary-sampling-method-row').remove();
+            event.stopPropagation();
```

Comment:
I tried to get this to work, but there was some issue preventing it (though at the moment, I can't remember what that issue was). I do remember it being exceedingly frustrating, because stopPropagation should have worked out of the box.

Reply:
It seems to work... at least the behavior seems the same as what is on production.